app/schemas/models/cinematic.schema.js

```diff
@@ -0,0 +1,151 @@
+const c = require('./../schemas')
+
+const CharacterSchema = (title) => c.object({
+  title: title,
+  description: 'ThangType that will appear on the right side of the screen.',
+  required: ['type']
+}, {
+  type: c.shortString({
+    title: 'Type',
+    description: 'Whether this is the player hero, a thangType slug or null.',
+    enum: ['slug', 'hero', 'null'],
+    default: 'null'
+  }),
+  slug: c.shortString({
+    title: 'ThangType Slug',
+    description: 'Required if type is set to `slug`'
+  }),
+  enterOnStart: {
+    type: 'boolean',
+    title: 'Animate in?',
+    description: 'If true the character will animate in. Otherwise the character will start simply there.'
+  },
+  position: c.point2d({ title: 'Position', description: 'Where character is located.' })
+})
+
+const ShotSetup = c.object({
+  title: 'ShotSetup'
+}, {
+  cameraType: c.shortString({
+    title: 'Camera Type',
+    description: 'The shot type',
+    enum: ['right-close', 'left-close', 'dual'],
+    default: 'dual'
+  }),
+  rightThangType: CharacterSchema('Right Character'),
+  leftThangType: CharacterSchema('Left Character'),
+  backgroundArt: c.object({
+    title: 'Background Art',
+    description: 'The art in the background of this shot.'
+  }, {
+    type: c.shortString({ enum: ['slug', 'null'] }),
+    slug: c.shortString({
+      title: 'Background path',
+      description: 'Path to the background asset'
```

Comment:
Is this valid only if the type is 'slug'? Also, what does 'slug' type mean - can the background be a thang which has a slug, or does it refer to an image asset path for the background?

Reply:
I don't know yet. This will need to be resolved once that feature is implemented. I think this is just a thangType. I can add a comment to say that this is unsure.

You're right. If this exists then it is always slug. Otherwise null.

Reply:
Yeah, a comment will be helpful. Thank you.